Runtime/Scripts/Loading/CustomHeaderDownloadProvider.cs

```diff
@@ -72,23 +72,22 @@ public CustomHeaderDownloadProvider(HttpHeader[] headers)
         }
 
         /// <inheritdoc />
-        public async Task<IDownload> Request(Uri url)
+        public async Task<IDownload> RequestAsync(Uri url)
         {
             var req = new CustomHeaderDownload(url, RegisterHttpHeaders);
             await req.WaitAsync();
             return req;
         }
 
         /// <inheritdoc />
-#pragma warning disable CS1998
-        public async Task<ITextureDownload> RequestTexture(Uri url, bool nonReadable, bool forceLinear)
+        public async Task<ITextureDownload> RequestTextureAsync(Uri url, bool nonReadable, bool forceLinear)
         {
-#pragma warning restore CS1998
 #if UNITY_WEBREQUEST_TEXTURE
-            var req = new CustomHeaderTextureDownload(url,nonReadable,RegisterHttpHeaders);
+            var req = new CustomHeaderTextureDownload(url, nonReadable, RegisterHttpHeaders);
             await req.WaitAsync();
             return req;
 #else
+            await Task.Yield();
```

Comment:
Yes, to avoid the warning of async method not using await in it. On the old renderer the solution was to disable the warning but I think this could be cleaner than doing that. This block of code was copied from the old renderer.

Reply:
Ok, it make sense to avoid the warning. The method should anyway handle that case.